src/Components/Topbar/Topbar.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
TopBar? Two words? :)

Reply:
the 'official' component naming is Top App Bar or in short: Top Bar. 
https://material.io/develop/web/components/top-app-bar/

Reply:
I think that @mprevide refers to this: if the term is composed by two words, Bar have to be write with upper case, just like in short, TopBar